src/rest/QuestionAnswerQuery.ts

```diff
@@ -0,0 +1,10 @@
+export interface IQuestionAnswerQuery {
```

Comment:
It's not `Query`, though.
It's the response from the search API that contains that info.

Suggestion: `IQueryAnswerResponse` ?